test/e2e/store_gateway_test.go

```diff
@@ -75,18 +75,19 @@ func TestStoreGateway(t *testing.T) {
 	series := []labels.Labels{
 		labels.FromStrings("a", "1", "b", "2"),
 	}
+
```

Comment:
I think we are missing test cases for our dedup filter to make sure resolution grouping works, right?

Reply:
Done